server/resources/startConfig.yaml

```diff
@@ -24,17 +23,17 @@ workloads:
   hello2:
     runtime: podman
     agent: agent_B
-    restart: true
+    restart: ALWAYS
     tags:
       - key: owner
         value: Ankaios team
     runtimeConfig: |
-      image: alpine:latest
-      commandArgs: [ "echo", "Hello Ankaios"]
+      commandOptions: [ "--entrypoint", "/bin/sh" ]
+      commandArgs: [ "-c", "echo 'Always restarted.'; sleep 2"]
```

Comment:
Maybe something went wrong here, but we are missing the image now.

Reply:
Image is there and I have tested the config. In addition, I have changed the [complete-state.md](https://github.com/eclipse-ankaios/ankaios/pull/217/files#diff-d3dcc4236b80a08cf8cbef837db11a33e7ff5c96324de4dcb23c7c53f7b15ef3) again to output the same as starting the server with [startConfig.yaml](https://github.com/eclipse-ankaios/ankaios/pull/217/files#diff-d880e481433364e6a9c81da84063e6e22e3e1a7e86cb955c8bf55a00af2a559c).